### PR TITLE
Refactor Mobile Navigation Component

### PR DESCRIPTION
**Pull request by varun13169**
Refactor Mobile Navigation Component

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good to me . You are good to go

File: components/rating/ratingPage.js [hunk 1]
```diff
@@ -3,9 +3,8 @@ import {initMobileMenu, addFocusBySubSecId} from "/initNavigation.js";
 // Add buttin listener for hamburger functionality
```
Comment (zoyatanweer) [inline components/rating/ratingPage.js]: it should be button instead of buttin

File: components/textUtils/textUtilsPage.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
-import {initMobileMenu, addFocusBySubSecId} from "/initNavigation.js";
+import {
+    initMobileMenu,
+    addFocusBySubSecId
+} from "/initNavigation.js";
 
 // Add buttin listener for hamburger functionality
```
Comment (zoyatanweer) [inline components/textUtils/textUtilsPage.js]: here too

**Review by zoyatanweer — APPROVED** [review]
code is good and clean. good to merge.

File: components/rating/ratingPage.js (hunk 1 above)
Comment (varun13169) [reply components/rating/ratingPage.js]: Copy / Paste Shenanigans.
Rectified.

File: components/textUtils/textUtilsPage.js (hunk 2 above)
Comment (varun13169) [reply components/textUtils/textUtilsPage.js]: Copy / Paste Shenanigans.
Rectified.

**Pull request merged**